project/src/components/host/host.tsx

```diff
@@ -1,3 +1,5 @@
+import {generateId} from '../../mocks/offers';
```

Comment:
```suggestion
import { generateId } from '../../mocks/offers';
```
где-то есть спейсы,где-то нету